packages/react-accordion/src/components/AccordionHeader/__snapshots__/AccordionHeader.test.tsx.snap

```diff
@@ -8,7 +8,6 @@ exports[`AccordionHeader renders a default state 1`] = `
   <button
     className=""
     disabled={false}
-    id="accordion-header-1"
```

Comment:
Is it expected that we don't have `id` there anymore?

Reply:
Yes. this is due to removal of `id` mechanism. Has definitely nothing to do with this PR, I'll try to remove this change